pkg/descheduler/utils/sorter/pod_test.go

```diff
@@ -130,7 +130,7 @@ func TestSortPods(t *testing.T) {
 		corev1.ResourcePods:   1,
 	}
 	SortPodsByUsage(pods, podMetrics, nodeAllocatableMap, resourceToWeightMap)
```

Comment:
Here you need to focus on it, because it will affect the sorting results. @eahydra

Reply:
This new ordering looks reasonable.